testSrc/unit/io/flutter/devtools/DevToolsUtilsTest.java

```diff
@@ -55,5 +55,10 @@ public void validDevToolsUrl() {
       generateDevToolsUrl(devtoolsHost, devtoolsPort, serviceProtocolUri, page, false, null),
       "http://127.0.0.1:9100/?ide=Android-Studio&uri=http%3A%2F%2F127.0.0.1%3A50224%2FWTFTYus3IPU%3D%2F#timeline"
     );
+
+    assertEquals(
+      generateDevToolsUrl(devtoolsHost, devtoolsPort, serviceProtocolUri, page, false, null, "3c3f41"),
```

Comment:
This color doesn't seem like quite the one we want as it is a bit grey. Taking a screenshot, the background color in a window such as the code editor or the dart analysis window in the light theme is 0xffffff.
<img width="288" alt="Screen Shot 2020-08-31 at 8 50 24 PM" src="https://user-images.githubusercontent.com/1226812/91793154-3a2b6680-ebcc-11ea-8776-e86fe32ced99.png">

Reply:
This is actually the dark mode color, I've figured out how to get the color from IntelliJ's defaults and light mode is indeed 0xffffff. Do you care about the test value?

Reply:
would be nice to test light mode as well if that is easy.

Reply:
It's not very meaningful since the hex code is passed into the function, but it's certainly easy!